examples/visual-tests/amp-story/amp-story-page-attachment.js

```diff
@@ -24,14 +24,10 @@ module.exports = {
     await page.tap('.i-amphtml-story-page-open-attachment-label');
     await page.waitFor(410);
   },
-  /**
-   * TODO(@ampproject/wg-stories): fix flaky test:
-   * https://percy.io/ampproject/amphtml/builds-next/8331061/changed/473805330
-   *
-   * 'open attachment UI element with link': async (page, name) => {
-   *   const screen = page.touchscreen;
-   *   await screen.tap(200, 240);
-   *   await page.waitFor('amp-story-page#page-2[active]');
-   * },
-   */
+   'open attachment UI element with link': async (page, name) => {
+    const screen = page.touchscreen;
+    await screen.tap(200, 240);
+    await page.waitFor('amp-story-page#page-2[active]');
+    await page.waitFor(1600);
```

Comment:
Waiting for a timeout is usually a code smell, especially given that Travis is slower than you'd expect - is there absolutely no way to do this without a timeout?

Reply:
Is there a way to wait for animations to complete?